Remove the `python-sdk` auto-generation (#6772)
hoprnet/hoprnet: README.md

This will in particular run `clippy` for the entire Rust codebase.

#### Generate the Python SDK

No Python SDK is available to connect to the HOPRd API. However, you can generate one using the [generate-python-sdk.sh](/scripts/generate-python-sdk.sh) script.

Prerequisites:

- swagger-codegen3
- build the repository to get the `hoprd-api-schema` generated

> The generated SDK will be available in the `/tmp/hoprd-sdk-python/` directory. You can modify the script to generate SDKs for different programming languages supported by swagger-codegen3.

Reviewer: ```suggestion
The generated SDK will be available in the `/tmp/hoprd-sdk-python/` directoryModify the script to generate SDKs for different programming languages supported by swagger-codegen3.
```
